perllib/FixMyStreet/DB/Result/Comment.pm

```diff
@@ -6,6 +6,7 @@ package FixMyStreet::DB::Result::Comment;
 
 use strict;
 use warnings;
+use HTML::Entities;
```

Comment:
We have a function in FixMyStreet::Template::html_filter that could be used here (only encodes the amp/gt/lt/'/" rathe than all entities, which means you don't need to worry whether it does the right thing for non-ASCII; it might do, I just haven't checked!).